### PR TITLE
Detect injected substrate wallets

### PR DESCRIPTION
**Pull request by Nikita-Polyakov**
### **Description**
- **What does this PR do?**
  - Predefined extensions list is shortened to **Fearless**, **Polkadot**
  - Added functionality to detect _installed_ extensions
  
- **Why is this change needed?**
  We showed only predefined extensions: Fearless, Polkadot.js, SubWallet, Talisman.
  We want to show **Fearless** as _predefined_ & _recommended_, and **Polkadot.js** as _predefined_ extensions.
  Also, if user has another installed extensions, he should be able to connect account using them - app should detect installed extension (new functionality)

### **Type of change**
- [ ] Bug fix
- [x] New feature
- [ ] Refactor (cleaning up code without changing functionality)
- [ ] Documentation update
- [ ] Tests
- [ ] Other (describe below):

---

### **Checklist**
- [x] Code adheres to coding guidelines and standards.
- [x] Linting and formatting checks have passed, no local issues.
- [x] Existing tests pass successfully, no local errors.
- [x] Pull request is linked to a relevant issue or task.
- [ ] Tests are written for new features or fixes.
- [ ] Documentation has been updated (if applicable).
  
---

### **Related Issue/Task**
- Issue/Task link: [[Wallet] Show predefined & injected extensions to connect](https://soramitsu.atlassian.net/browse/PW-1746)

---

### **Testing**
- **How has this been tested?**
  - _Describe manual testing steps or link to relevant automated tests._
  
- **Edge cases covered:**
  - _List any specific cases handled by this PR._

---

### **Screenshots/Video/Link to dedicated environment (if applicable)**
extensions

---

### **Additional Notes**
- _Add any information the reviewer should consider, especially cross-project impacts, UX changes, or potential risks._

---

### **Reviewer Tasks**
- [x] Review logic for correctness and clarity.
- [x] Verify that the code is following best practices.
- [ ] Test or review related areas if this PR affects other projects.

### CODE REVIEW

File: src/components/Connection/Step/ExtensionList.vue [hunk 1]
```diff
@@ -49,6 +56,9 @@ export default class ExtensionListStep extends Mixins(TranslationMixin) {
   @Prop({ default: false, type: Boolean }) readonly selectedWalletLoading!: boolean;
   @Prop({ default: () => [], type: Array }) readonly internalWallets!: Wallet[];
   @Prop({ default: () => [], type: Array }) readonly externalWallets!: Wallet[];
+  @Prop({ default: () => [], type: Array }) readonly recommendedWallets!: string[];
+
+  readonly wikiLink = 'https://wiki.sora.org/polkaswap-connect-wallet.html';
```
Comment (Kron1749) [inline src/components/Connection/Step/ExtensionList.vue:61]: Move to consts?

**Comment by bertikaz** [comment]
Please have a look on broken pipes as well
